server/src/main/java/org/opensearch/indices/EhcacheDiskCachingTier.java

```diff
@@ -52,7 +56,7 @@ public class EhcacheDiskCachingTier implements DiskCachingTier<IndicesRequestCac
     private final String diskCacheFP; // the one to use for this node
     private RemovalListener<IndicesRequestCache.Key, BytesReference> removalListener;
     private ExponentiallyWeightedMovingAverage getTimeMillisEWMA;
-    private static final double GET_TIME_EWMA_ALPHA  = 0.3; // This is the value used elsewhere in OpenSearch
+    private static final double GET_TIME_EWMA_ALPHA = 0.3; // This is the value used elsewhere in OpenSearch
```

Comment:
```ExponentiallyWeightedMovingAverage(GET_TIME_EWMA_ALPHA, 10)```. Keeping 10 as intialAvg doesn't seem right. Don't know the right value but should keeping it 0 be better?

Reply:
I somewhat arbitrarily picked 10 since we expect a disk seek to take ~10 ms on a spinning disk but 0 might be better, yeah